ducktape/tests/status.py

```diff
@@ -30,3 +30,5 @@ def to_json(self):
 PASS = TestStatus("pass")
 FAIL = TestStatus("fail")
 IGNORE = TestStatus("ignore")
+OPASS = TestStatus("opass")
```

Comment:
For readability purposes, please rename into something like FLAKY_PASS and FLAKY_FAIL

Reply:
edit: UNSTABLE_PASS and UNSTABLE_FAIL